.github/workflows/cicd.yml

```diff
@@ -6,16 +6,46 @@ jobs:
     runs-on: ubuntu-20.04
     strategy:
       matrix:
-        python-version: [3.8]
-        tox-env:
-          - py3-django2
-          - py3-django3
-          - py3-django4
-          - bandit
-          - black
-          - flake8
-          - isort
-          - mypy
+        include:
+          - name: bandit
+            python-version: 3.8
+            tox-env: bandit
+          - name: black
+            python-version: 3.8
+            tox-env: black
+          - name: flake8
+            python-version: 3.8
+            tox-env: flake8
+          - name: isort
+            python-version: 3.8
+            tox-env: isort
+          - name: mypy
+            python-version: 3.8
+            tox-env: mypy
+          - name: python3.5 django2
+            python-version: 3.5
+            tox-env: py3-django2
+          - name: python3.6 django2
+            python-version: 3.6
+            tox-env: py3-django2
+          - name: python3.6 django3
+            python-version: 3.6
+            tox-env: py3-django3
+          - name: python3.8 django2
+            python-version: 3.8
+            tox-env: py3-django2
+          - name: python3.8 django3
+            python-version: 3.8
+            tox-env: py3-django3
+          - name: python3.8 django4
+            python-version: 3.8
+            tox-env: py3-django4
+          - name: python3.10 django3
+            python-version: '3.10'
```

Comment:
Fun fact if you haven't encountered it yet: github actions interprets the python version as a floating point number by default. If you don't include the quotes, it tries to run it on python 3.1 since it thinks that that is equivalent to 3.10.

I feel like JavaScript is somehow to blame...

Reply:
good to know - thanks for sharing!